styleguide/screens/Patterns/TabBar/TabBar.js

```diff
@@ -12,7 +12,7 @@ import TabBarIconItem from '../../../../components/Navigation/TabBar/TabBarIconI
 
 import m from '../../../../globals/modifiers.css';
 
-const BadgeDocumentation = () => (
+const TabBarDocumentation = () => (
```

Comment:
Oops (I'm awful for this)

Reply:
I definitely did not notice it in the PR 😆